torch/_inductor/codegen/triton.py

```diff
@@ -385,6 +385,12 @@ def _print_TruncToInt(self, expr):
             f"libdevice.trunc({self._print(expr.args[0])}).to({V.kernel.index_dtype})"
         )
 
+    def _print_Float(self, expr):
```

Comment:
I admit I don't know if it's always valid to assume constants are float64. I'm operating on the assumption that any float literal originated from Python and is technically float64.